addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8026,7 +8026,11 @@ msgctxt "#15064"
 msgid "Apply changes directly"
 msgstr ""
 
-#empty string with id 15065
+#. Warning if no dsp modes are present
+#: xbmc/settings/dialogs/GUIDialogAudioDSPManager.cpp
+msgctxt "#15065"
+msgid "No audio DSP modes present!"
```

Comment:
Yes, I think "available" is better. Wouldnt it be possible to not open the manager dialog at all then?